match/match.py

```diff
@@ -150,7 +150,7 @@ async def addMatches(self, ctx, *matches):
 
         Examples:
 
-        [p]addMatches "['1','September 10, 2018','Fire Ants','Leopards',
+        [p]addMatches "['1','September 10, 2020','Fakes','Leopards',
```

Comment:
Is there a reason this was changed? I like having the example show an example of a team name with multiple words to show how that can be handled inside the quotes.

Reply:
Nah I was just playing around with it. I'm happy to change it back.